### PR TITLE
feat: solana create account coin support

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6584 Add Solana create account on coin support and CLI](https://app.clickup.com/t/9002019994/PRF-6584)

File: apps/cli/src/config.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const config = {
   API_CYPHEROCK: 'https://dev-api.cypherock.com',
   ALLOW_PRERELEASE: true,
-  LOG_LEVEL: 'error',
+  LOG_LEVEL: 'debug',
```
Comment (irshadCypherock) [inline apps/cli/src/config.ts]: We can revert this

File: packages/coin-support-solana/package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "build:cjs": "tsc -p tsconfig_cjs.json",
     "build:esm": "tsc -p tsconfig.json",
     "build:dirty": "pnpm build:esm",
+    "test": "jest --passWithNoTests",
```
Comment (irshadCypherock) [inline packages/coin-support-solana/package.json]:
```suggestion
    "test": "jest",
```

File: packages/coin-support-solana/src/services/api/wallet.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+export const getBalance = async (address: string, assetId: string) => {
+  const url = `${baseURL}/balance`;
+  const response = await makePostRequest(url, {
+    address,
+    network: solanaCoinList[assetId].network,
+    responseType: 'v2',
+  });
+
+  let { balance } = response.data ?? '0';
```
Comment (irshadCypherock) [inline packages/coin-support-solana/src/services/api/wallet.ts]:
```suggestion
  let balance = response.data?.balance ?? '0';
```

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**